api/API_VERSIONING.md

```diff
@@ -84,52 +100,43 @@ from clients (Envoy, gRPC) and a large number of control planes, ranging from si
 management servers to xDS-as-a-service offerings run by vendors. The [xDS API
 shepherds](https://github.com/orgs/envoyproxy/teams/api-shepherds) will make the decision to add a
 new major version subject to the following constraints:
-* There exists sufficient technical debt in the xDS APIs in the existing supported major version
+* There exists a major, breaking change, to the xDS APIs in the existing supported major version
   to justify the cost burden for xDS client/server implementations.
 * At least one year has elapsed since the last major version was cut.
 * Consultation with the Envoy community (via Envoy community call, `#xds` channel on Slack), as
   well as gRPC OSS community (via reaching out to language maintainers) is made. This is not a veto
   process; the API shepherds retain the right to move forward with a new major API version after
   weighing this input with the first two considerations above.
 
-Following the release of a new major version, the API lifecycle follows a deprecation clock.
```

Comment:
I don't think we should delete this yet, see above comment. Instead, create a subsection header for major version clock and another for minor version (which is what you're adding below).